README.md

```diff
@@ -48,13 +48,24 @@ export class HomeComponent {
     fromEvent(document, 'mousemove').subscribe()
   ];
 
-  // You can still use the opertator
+  // You can still use the operator
   ngOnInit() {
     interval(1000).pipe(untilDestroyed(this));
   }
 }
 ```
 
+You can set the `blackList` property if you **don't** want to unsubscribe to one or more subscriptions
+
+```ts
+@UntilDestroy({ blackList: ['actionsSubject'] })
+@Component({})
+export class HomeComponent {
+  // actionsSubject will not be unsubscribe upon component destruction
+  constructor(private actionsSubject: ActionsSubject)
```

Comment:
```suggestion
  // actionsSubject will not be unsubscribed upon component destruction
  constructor(private actionsSubject: ActionsSubject) {}
```

Could you provide a full example, please? I would also avoid injecting class that extends subject, I foresee misunderstanding on the part of users

Let it be a super-simple example, like `subject$ = new Subject()` or something else.

Reply:
I agree with you... it makes sense to have something simpler.
I added some code that was similar to the tests. Let me know if that works!